cmd/zetaclientd/start.go

```diff
@@ -217,12 +219,24 @@ func Start(_ *cobra.Command, _ []string) error {
 		return errors.Wrap(err, "unable to create chain observer map")
 	}
 
+	cometbftURL := fmt.Sprintf("http://%s:%d", cfg.ZetaCoreURL, 26657)
```

Comment:
Consider adding this as a function for Config struct.

Reply:
I'd prefer to keep this very simple for now and refactor it later. We also instantiate this client internally in zetaclient/zetacore/client_cosmos.go but it really should probably be injected from the top imo.

Reply:
Can we open an issue for the future consideration?